assets/hip-1056/protobuf/stream/output/token_service.proto

```diff
@@ -171,10 +171,33 @@ message UpdateTokenNftsOutput {}
  * the original transaction.
  */
 message TokenAirdropOutput {
-  /**
-   * Custom fees assessed during a TokenAirdrop.
-   * <p>
-   * These fees SHALL be present in the full transfer list for the transaction.
-   */
-  repeated proto.AssessedCustomFee assessed_custom_fees = 1;
+    /**
+    * Custom fees assessed during a TokenAirdrop.
+    * <p>
+    * These fees SHALL be present in the full transfer list for the transaction.
+    */
+    repeated proto.AssessedCustomFee assessed_custom_fees = 1;
+
+    /**
+     * A list of token associations.
+     * <p>
+     * This field SHALL list all token associations created automatically
+     * while handling this transaction.
+     */
+    repeated proto.TokenAssociation automatic_token_associations = 2;
+
+    /**
+     * A list of _non-HBAR_ token transfers, in single-entry form.
+     * <p>
+     * This SHALL include all _non-HBAR_ token transfers completed as a
+     * result of this transaction.<br/>
+     * This MUST include, at least,
+     * <ul>
+     *   <li>Each source and recipient of custom fees</li>
+     *   <li>All transfers directly performed by this transaction</li>
+     *   <li>Any transfers performed by a smart contract call associated
+     *       with this transaction</li>
+     * </ul>
+     */
+    repeated proto.TokenTransferList token_transfer_lists = 3;
```

Comment:
We need to check this.  I recall something about token transfers being possible in a fairly wide range of transactions (which is why it ended up in the transaction result).

Reply:
I believe the newest note was `TransactionResult` should only contain items that are applicable to all transactions, not some (even if the majority). You also noted that filtering logic wouldn't work well if this isn't adhered too.
Is there flexibility on this?

Reply:
I was more noting that we may need to add the token transfer lists to a few more places.